apollo-portal/src/main/config/application-ldap.yml

```diff
@@ -0,0 +1,21 @@
+spring:
```

Comment:
how about renaming this file to `application-ldap-apacheds-sample.yml` so that we could also add other ldap sample configuration files such as openldap, active directory, etc.

Reply:
我这里只有openldap 只加了application-ldap-apacheds-sample.yml和application-ldap-openldap-sample.yml两个基础配置